grub-core/kern/efi/tpm.c

```diff
@@ -161,21 +161,12 @@ grub_tpm_execute(PassThroughToTPM_InputParamBlock *inbuf,
   }
 }
 
-typedef struct {
-	grub_uint32_t pcrindex;
-	grub_uint32_t eventtype;
-	grub_uint8_t digest[20];
-	grub_uint32_t eventsize;
-	grub_uint8_t event[1];
-} Event;
```

Comment:
I think this deletion should be in the same commit with changing `event` to `TCG_PCR_EVENT` below since they seem to be parts of the same change (dropping a redundant struct).  That can be in a separate commit from the `buf` casting change.  This is minor, though, and I'm okay with merging this without it.